mobile/cmp/navigator/NavigatorModel.js

```diff
@@ -48,13 +51,16 @@ export class NavigatorModel extends HoistModel {
     /**
      * @param {Object[]} pages - configs for PageModels, representing all supported
      *      pages within this Navigator/App.
+     * @param {boolean} [track] - True to enable activity tracking of page views (default false).
```

Comment:
Re: comment above: could be a boolean or config, and if set to true, will take the default {oncePerSession: true} flag